notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -556,56 +636,566 @@
+   "source": [
+    "Let's see correction results. The generated texts for quantized INT8 model and original FP32 model should be almost the same."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "86d39904-21a8-4125-bb1d-1785aaadd85a",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-09-27T12:37:11.035199500Z",
+     "start_time": "2023-09-27T12:37:08.172901100Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "20474adbb06040d0b3699a00af09d131",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "correcting text..:   0%|          | 0/1 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Input text:                   Most of the course is about semantic or  content of language but there are also interesting topics to be learned from the servicefeatures except statistics in characters in documents.At this point, He introduces herself as his native English speaker and goes on to say that if you contine to work on social scnce\n",
+      "\n",
+      "Generated text by INT8 model: Most of the course is about the semantic content of language but there are also interesting topics to be learned from the service features except statistics in characters in documents. At this point, she introduces himself as a native English speaker and goes on to say that if you continue to work on social issues, you will continue to be successful.\n"
+     ]
+    }
+   ],
+   "source": [
+    "if to_quantize.value == \"Enable\":\n",
+    "    corrected_text_int8 = correct_text(default_text, grammar_checker_pipe, grammar_corrector_pipe_int8)\n",
+    "    print(f\"Input text:                   {default_text}\\n\")\n",
+    "    print(f'Generated text by INT8 model: {corrected_text_int8}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "176da8b1",
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "source": [
+    "### Compare model size, performance and accuracy [$\\Uparrow$](#Table-of-contents:)\n",
+    "\n",
+    "First, we compare file size of `FP32` and `INT8` models."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "e8277b8b",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-09-27T12:37:11.039089700Z",
+     "start_time": "2023-09-27T12:37:11.038799100Z"
+    },
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Model: openvino_decoder_with_past_model\n",
+      "    * FP32 IR model size: 1658150.26 KB\n",
+      "    * INT8 IR model size: 415713.38 KB\n",
+      "    * Model compression rate: 3.989\n"
```

Comment:
It would be better to have " Model compression rate:" below where you report acceleration and accuracy drop. I would also call it "Model footprint reduction: 3.989x"

Reply:
Done